code-scanning/properties/sonarcloud.properties.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "SonarCloud",
+    "creator": "Sonar",
+    "description": "Static analysis of code for vulnerability detection, covering 26+ languages. Free for open source projects",
```

```suggestion
    "description": "Static analysis of code for vulnerability detection, covering 26+ languages.",
```

I would refrain from using `free`/`paid` or talking about plans here. You can do that by putting a comment in workflow file.

---

Rather call out any quick USP which Sonar  cloud may provide.

---

@christophe-havard-sonarsource Can you please address this as well? Thanks

---

done